src/components/App.tsx

```diff
@@ -349,6 +359,7 @@ class App extends React.Component<AppProps, AppState> {
     }
     return (
       <canvas
+        className="excalidraw__canvas"
```

Comment:
Don't we want to start using BEM from now on?

Reply:
that's why I named it `excalidraw__canvas` => `block__element`, what do you suggest ?

Reply:
Okay, I think my BEM game isn't strong. I thought it only uses dashes :).